### PR TITLE
Lesson 2

### PR DESCRIPTION
**Pull request by bramakrishna16**

### CODE REVIEW

File: fileSys.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const fs= require('fs');
+
+const fileSys=(err,files)=>{
+  files.forEach(printOtherNames=(ele)=>{
+    if (ele != 'bharath'){
+      fs.writeFile('/home/bharath/gitcode/garagescript/files.txt',files+ "\n");
```
Comment (songz) [inline fileSys.js]: this will give you a file with only one person (the last one). Please think about this more.

File: fileSys.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const fs= require('fs');
+
+const fileSys=(err,files)=>{
+  files.forEach(printOtherNames);
+}
+const printOtherNames=(eachName)=>{
+  if (eachName != 'bharath'){
+    fs.appendFile('/home/bharath/gitcode/garagescript/files.txt',eachName+ "\n");
```
Comment (songz) [inline fileSys.js]: can you try doing it with just writeFile?
hint* build a string, then after you have a string, call the writeFile function

File: fileSys.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+const fs= require('fs');
+
+const fileSys=(err,files)=>{
+   let names="";
+   files.forEach((eachName)=>{
+    if (eachName != 'bharath'){
+      names = names+  ' ' + eachName;
+    }
+  });
+   fs.writeFile('/home/bharath/gitcode/garagescript/files.txt',names+ "\n");
```
Comment (saumya665) [inline fileSys.js]: fs takes in a third parameter as well

File: fileSys.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+const emptyFunc=()=>{
+}
+const fileSys=(err,files)=>{
+   let names="";
+   files.forEach((eachName)=>{
+    if (eachName != 'bharath'){
+      names = names+  ' ' + eachName;
+    }
+  });
+   fs.writeFile('/home/bharath/gitcode/garagescript/files.txt',names,emptyFunc);
```
Comment (saumya665) [inline fileSys.js]: I think the way Song explained it, you should be able to directly pass in ()=>{} as the third parameter in 
writeFile

Comment (saumya665) [inline fileSys.js]: saves some lines of code and helps simpliy

**Pull request merged**